modules/monitor/src/components/Monitor.vue

```diff
@@ -107,100 +120,109 @@
           </tr>
           <tr>
             <td>Profile</td>
-            <td>{{ selectedProcess.UserProfileKey }}</td>
+            <td>{{ selectedProcess.LocalProfileKey }}</td>
           </tr>
         </tbody>
       </table>
 
       <div v-if="selectedProcess._childProcesses">
-        <h4>childProcesses:</h4>
-        <ChildProcessList v-bind:processes="selectedProcess._childProcesses"></ChildProcessList>
+        <h4>Child Processes:</h4>
+        <ChildProcessList :processes="selectedProcess._childProcesses"></ChildProcessList>
       </div>
 
+      <div class="debugging">
+        <h3>Debugging <small>...left here intentionally, for now.</small></h3>
+        <PRE>{{ selectedProcess | clean_object }}</PRE>
+      </div>
     </div>
-    <div v-else-if="selected == 2 && !selectedCommunication._deleted" class="eleven wide column container content-pane">
-      <h2>{{ selectedCommunication._parent.Name }} ({{ selectedCommunication._parent.Pid }}) >
-        <span v-if="selectedCommunication.Domain == 'IH'">Incoming from Localhost</span>
-        <span v-else-if="selectedCommunication.Domain == 'IL'">Incoming from the LAN</span>
-        <span v-else-if="selectedCommunication.Domain == 'II'">Incoming from the Internet</span>
-        <span v-else-if="selectedCommunication.Domain == 'IX'">Incoming - Invalid</span>
-        <span v-else-if="selectedCommunication.Domain == 'PH'">Peers on Localhost</span>
-        <span v-else-if="selectedCommunication.Domain == 'PL'">Peers on the LAN</span>
-        <span v-else-if="selectedCommunication.Domain == 'PI'">Peers on the Internet</span>
-        <span v-else-if="selectedCommunication.Domain == 'PX'">Peers - Invalid</span>
-        <span v-else>{{ selectedCommunication.Domain }}</span>
+    <div v-else-if="selected == 2" class="eleven wide column container content-pane">
+      <h2>
+        {{ selectedScope._process.Name }} ({{ selectedScope._process.Pid }}) >
+        <span v-if="selectedScope.name == 'IH'">Incoming from Localhost</span>
+        <span v-else-if="selectedScope.name == 'IL'">Incoming from the LAN</span>
+        <span v-else-if="selectedScope.name == 'II'">Incoming from the Internet</span>
+        <span v-else-if="selectedScope.name == 'IX'">Incoming - Invalid</span>
+        <span v-else-if="selectedScope.name == 'PH'">Peers on Localhost</span>
+        <span v-else-if="selectedScope.name == 'PL'">Peers on the LAN</span>
+        <span v-else-if="selectedScope.name == 'PI'">Peers on the Internet</span>
+        <span v-else-if="selectedScope.name == 'PX'">Peers - Invalid</span>
+        <span v-else>{{ selectedScope.name }}</span>
       </h2>
```

Comment:
That code is duplicated, isn't it possible to render a `<template>` at multiple places? (disclaimer, I don't really know Vue yet)